cf-check/db_structs.h

```diff
@@ -0,0 +1,152 @@
+#ifndef CFENGINE_DB_STRUCTS_H
+#define CFENGINE_DB_STRUCTS_H
+
+// All structs needed for cf-check pretty printing
+// Structs from libutils are included normally via headers
+// Structs from libpromises are duplicated; cf-check doesn't use libpromises
+
+#include <statistics.h>
+// typedef struct
+// {
+//     double q;
+//     double expect;
+//     double var;
+//     double dq;
+// } QPoint;
```

Comment:
Why is this one commented out?

Reply:
Because it's in statistics.h , which is in libutils and can be included.